app/src/main/java/com/eveningoutpost/dexdrip/UtilityModels/CollectionServiceStarter.java

```diff
@@ -116,6 +116,9 @@ private static void processPending() {
     public static boolean isFollower(Context context) {
         return Pref.getString("dex_collection_method", "").equals("Follower");
     }
+    public static boolean isLibre2App(Context context) {
```

Comment:
Are any of the changes in this class required?

Reply:
yes for limitting the Calibrations (newer commit). If the DoNothingService is removed, private static boolean isLibre2App(String collection_method) can be removed too

Reply:
Calibrations are now in my repo too: https://github.com/SandraK82/xDrip-plus/commit/e3e13292f3319aaa7b8b37e1e29d7c593c073e0f